telescoop_backup/backup.py

```diff
@@ -65,14 +71,18 @@ def backup_file(file_path: str, remote_key: str, connexion=None, skip_if_exists=
         except connexion.exceptions.ClientError as e:
             if e.response["Error"]["Code"] != "404":
                 raise
-
     connexion.upload_file(file_path, BUCKET, remote_key)
 
 
 def backup_folder(path: str, remote_path: str, connexion=None):
     """Recursively backup entire folder. Ignores paths that were already backup up."""
     if connexion is None:
         connexion = boto_client()
+    number_of_files = sum([len(files) for root, dirs, files in os.walk(path)])
+    if number_of_files > 100:
```

Comment:
pas d'accord, pour des backups quotidien (comme c'est le cas la majorité du temps) c'est bcp plus optimisé et simple de ne pas zipper pour ne pas sauvegarder plusieurs fois le même fichier. Je supprimerais juste.

Reply:
C'est juste un `print`, ça ne change pas le comportement donc je propose de garder en changeant plutôt le message en "You may want to use `--zipped` option." ?